### ISSUE TITLE
[Bug]  Crash when picking up shulkerbox

### ISSUE THREAD

**Issue by energy11a** [MeteorDevelopment/meteor-client#2024]
### Describe the bug

Crashes when picking up shulkerbox with too many slots
[crash-2022-02-06_09.44.00-client.txt](https://github.com/MeteorDevelopment/meteor-client/files/8009681/crash-2022-02-06_09.44.00-client.txt)

### Steps to reproduce

```bash
/give @p shulker_box{BlockEntityTag:{Items:[{Slot:0,id:totem_of_undying,Count:127}]}} 1
turn better tooltips on
place shulkerbox
break in survival
collect shulkerbox
Crash
```

### Meteor Version

1427

### Minecraft Version

1.18.1

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by Wide-Cat** [comment]
can you post a video of you replicating this? I wasn't able to

**Comment by bluepanee** [comment]
> can you post a video of you replicating this? I wasn't able to

I can confirm this. Toggle "Open with middle click" and just click on this shulker.
You will get this:
```Java
java.lang.ArrayIndexOutOfBoundsException: Index 27 out of bounds for length 27
	at meteordevelopment.meteorclient.utils.Utils.getItemsInContainerItem(Utils.java:194)
	at meteordevelopment.meteorclient.utils.Utils.openContainer(Utils.java:169)
	at net.minecraft.class_465.handler$zme000$mouseClicked(class_465.java:1267)
```

**Comment by Sersoid** [comment]
I got exactly the same crashlog when getting this shulker into inventory with ContainerViewer enabled in HUD

**Comment by energy11a** [comment]
https://github.com/energy11a/meteor-crash/upload/main

**Comment by MineGame159** [comment]
Fixed in bc55567d6c93ff4a0e55bd3608a7b83d6993a84e.
